sportngin/homebrew-homebrew
# RKT-8764 More depricated dependencies

What
----------------------
Fix unsupported java dependency specification.   This time get rid of the formula since it is no longer used.

Why
----------------------
Same reason we had to do https://github.com/sportngin/homebrew-homebrew/pull/16

Deploy Plan
-----------
> Does Platform Operations need to know anything special about this deploy? Are migrations present?

Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

URLs
----
> Links to bug tickets or user stories.

QA Plan
-------
> Reference commonly used Regression test plans on the [QA Wiki](https://github.com/sportngin/qa-tests/wiki)
> Fill in scenarios below in checklist format.
> Consider Regression scenarios (did we break something else related to this change) in addition to Happy Path (testing the new feature directly).
> Evaluate the risk level and label accordingly and ensure the QA Plan matches the risk level!

- [x] to get testing, do:
```
export HOMEBREW_DEVELOPER=1
brew tap --full sportngin/homebrew-homebrew
cd `brew --repository sportngin/homebrew-homebrew`
git checkout RKT-8764_more_depricated_dependencies  
```
- [x] brew update succeeds
- [x] `>~/sportsengine > ./setup`  succeeds
- [x] get back to life as normal:
```
git checkout master
unset HOMEBREW_DEVELOPER
```
